core/domain/services/admin/privacy/policy/privacy_policy.template.php

```diff
@@ -0,0 +1,72 @@
+<h2><?php esc_html_e('Event Registration Data', 'event_espresso');?></h2>
+<p><?php esc_html_e('We collect information about you during event registration. This information may include but is not limited to:', 'event_espresso');?></p>
+<ul>
+    <li><?php esc_html_e('Your names', 'event_espresso');?></li>
+    <li><?php esc_html_e('Billing address', 'event_espresso');?></li>
+    <li><?php esc_html_e('Mailing address', 'event_espresso');?></li>
+    <li><?php esc_html_e('Email address', 'event_espresso');?></li>
+    <li><?php esc_html_e('Phone number', 'event_espresso');?></li>
+    <li><?php esc_html_e('Location and traffic data (including IP address and browser type)', 'event_espresso');?></li>
+    <li><?php esc_html_e('Any other details that might be requested from you for the purpose of processing your registration or ticket purchase', 'event_espresso');?></li>
+</ul>
+
+<p><?php esc_html_e('Handling this data also allows us to:', 'event_espresso');?></p>
+<ul>
+    <li><?php esc_html_e('Send you important account/purchase/service information.', 'event_espresso');?></li>
+    <li><?php esc_html_e('Respond to your queries, refund requests, or complaints.', 'event_espresso');?></li>
+    <li><?php esc_html_e('Process payments and to prevent fraudulent transactions. We do this on the basis of our legitimate business interests.', 'event_espresso');?></li>
+    <li><?php esc_html_e('Set up and administer your account, provide technical and customer support, and to verify your identity.', 'event_espresso');?></li>
+</ul>
+
+<?php if (!empty($active_onsite_payment_methods) || !empty($active_offsite_payment_methods)) { ?>
+<h2><?php esc_html_e('Billing Information', 'event_espresso');?> </h2 >
+<?php
+// if onsite or offsite payment methods are active
+if (! empty($active_onsite_payment_methods)) { ?>
+    <p><?php esc_html_e('In order to process payments, we collect billing information on-site. Sensitive billing information is not stored on our server, but may be handled while in-transit to the payment processing server.',
+            'event_espresso'); ?></p>
+    <p><?php printf(
+            esc_html__('Please see the privacy policy of %1$s.', 'event_espresso'),
```

Comment:
plz use `esc_html_x()` in order to provide context
(duplicate the actual string but swap placeholders with example content)
ex: `esc_html_x('attendee #%1$d', 'attendee #123', 'event_espresso')`